Source/Blazorise/Utils/CompositeDisposable.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.ObjectModel;
+
+namespace Blazorise.Utils
+{
+    public sealed class CompositeDisposable : Collection<IDisposable>, IDisposable
+    {
+        public void Dispose()
+        {
+            foreach ( var disposable in Items )
+            {
+                disposable.Dispose();
+            }
+
+            Clear();
+        }
+    }
+
+    public static class CompositeDisposableEx
```

Comment:
I like this pattern, and will borrow it :) Maybe the naming should be better so it's more clear what it's doing?

Reply:
The pattern of `DisposableAction` and `CompositeDisposable` comes from System.Reactive so it is more or less wellknown in the C# world. I put it in the central Utils namespace so it can be used everywhere